modmesh/core.py

```diff
@@ -64,6 +64,7 @@
     'METAL_BUILT',
     'metal_running',
     'HAS_VIEW',
+    'Gmsh',
```

Comment:
This is a class and should be put before constants, ordered in the same convention as other classes or functions.